netmaster/mastercfg/endpointGroupState.go

```diff
@@ -34,7 +34,7 @@ type EndpointGroupState struct {
 	PktTagType      string `json:"pktTagType"`
 	PktTag          int    `json:"pktTag"`
 	ExtPktTag       int    `json:"extPktTag"`
-	EpCount         int    `json:"epCount"`
+	EpCount         int    // To store endpoint Count
```

Comment:
you probably want to keep the `json:"epCount"` annotation and add the comment after the annotation